carma_cloud_client/src/main.cpp

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (C) 2022 LEIDOS.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+#include <rclcpp/rclcpp.hpp>
+#include "carma_cloud_client/carma_cloud_client_node.hpp"
+
+int main(int argc, char **argv) 
+{
+  std::cerr<<"0"<<std::endl;
```

Comment:
Why do we need the 0-1-2 error here? Can we catch the error if any and provide detailed description?

Reply:
those were debugging code, should have been removed